babel.config.js

```diff
@@ -1,7 +1,13 @@
-
+// for ESM don't transpile modules
```

Comment:
This might bite us later down the line with TypeScript?

Reply:
needed for tree shaking, and besides we still have a cjs build at the same path